doc/tutorials/focas_howto.rst

```diff
@@ -0,0 +1,484 @@
+four tabs in the `ginga`_ window:
+
+.. image:: ../figures/focas_sciimg.png
+   :scale: 60%
+   :align: center
+.. image:: ../figures/focas_skyres.png
+   :scale: 60%
+   :align: center
+.. image:: ../figures/focas_res.png
+   :scale: 60%
+   :align: center
+
+This shows on the top the calibrated science image, in the middle the sky-subtracted
+calibrated image, and on the bottom the sky residual image (sky-subtracted
+calibrated image divided by the uncertainties).
+The green/magenta lines are the slit edges.  The orange lines (shown only in the first channel)
+are the object traces. 
+See :ref:`spec-2d-output` for further details.
+
+The main assessments to perform are to make sure that the object is well traced,
+that there are little to no strong sky residuals in the ``sky_resid`` channel,
+and that the data in the ``resid`` channel looks like pure noise (see also :ref:`pypeit_chk_noise_2dspec`).
+
+
+Spec1D
+------
+
+You can see a summary of all the extracted sources in the ``spec1d*.txt`` files saved
+in the ``Science/`` folder.  For this example, here are the first few lines of the file
+``Science/spec1d_FCSA00216334-SN2019muj_FOCAS_20201121T083826.517.txt``:
+
+.. code-block:: console
+
+    | slit |                    name | obj_id | spat_pixpos | spat_fracpos | box_width | opt_fwhm |  s2n | wv_rms |
+    |  760 | SPAT0634-SLIT0760-DET01 |    634 |       634.2 |        0.249 |      3.00 |    0.805 | 2.14 |  0.231 |
+    |  760 | SPAT0700-SLIT0760-DET01 |    700 |       699.6 |        0.380 |      3.00 |    3.944 | 1.37 |  0.231 |
+
+
+It shows a table with the PypeIt names of the extracted spectra in each slit and all the associated
+information about the extraction and the object. See :ref:`spec1d-extract_info` for a detailed description of this file.
+**Note that the maskdef_id, objname, objra, objdec and maskdef_extract columns are not
+printed for long-slit observations.**
+
+To inspect the 1D spectrum, we can use the script :ref:`pypeit_show_1dspec`, with a call like this:
+
+.. code-block:: bash
+
+    pypeit_show_1dspec Science/spec1d_FCSA00216334-SN2019muj_FOCAS_20201121T083826.517.fits --ginga
+
+which plots the spectrum in a tab of the `ginga`_ viewer and allows to select the different extracted spectra using a
+drop down menu, in addition to selecting other properties of the spectrum. Here is one exemple:
+
+.. figure:: ../figures/focas_spec1d_ginga.png
+   :scale: 60%
+   :align: center
+
+The black line is the flux and the red line is the estimated error. In the `ginga`_ window, the user can
+select the different extracted spectra, the extraction type (``OPT`` or ``BOX``), fluxed or not fluxed spectrum, and
+if showing masked data, by using the drop down menu on the right of the window. Common spectral features
+can be over-plotted by selecting an option in the ``Line lists`` drop down menu and providing a redshift.
+
+See :ref:`spec-1d-output` for further details.
+
+
+..  REMOVE?
```

Comment:
please remove this one if you think it's not going to be included.

Reply:
gone